molo/core/content_import/tests/base.py

```diff
@@ -104,3 +115,42 @@ def create_workspace(self, **kw):
         })
 
         return ws
+
+    def create_category(self, *args, **kw):
+        [cat] = self.create_categories(*args, count=1, **kw)
+        return cat
+
+    def create_page(self, *args, **kw):
+        [page] = self.create_pages(*args, count=1, **kw)
+        return page
+
+    def add_languages(self, ws, *locales):
+        for locale in locales:
+            lang = eg_models.Localisation({'locale': locale})
+            ws.save(lang, 'Added %s' % locale)
+
+    def catch(self, error_cls, fn):
+        error = None
+
+        try:
+            fn()
+        except error_cls as e:
+            error = e
+
+        self.assertTrue(
+            error is not None,
+            "Expected an error to be raised")
+
+        return error
+
+
+class Counter(object):
+    def __init__(self, i=0):
+        self.i = i
+
+    def next(self):
+        self.i = self.i + 1
+        return self.i
+
+
+index_counter = Counter(int(time()))
```

Comment:
Did you create an issue for this? I think there's a way to fix it, but will need to dig in